javascript/index.js

```diff
@@ -14,8 +14,12 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
-}
+    let myInterval = setInterval(function() {
```

Comment:
The chronometer already has an interval running in it's class (maybe two in case of bonus) so you want to use those to print the time. That is why you can use call backs here. Create a function to update the DOM and then pass it as a callback to be run inside the class interval.